en/restore-data-using-tidb-lightning.md

```diff
@@ -137,7 +137,9 @@ To restore backup data from the remote source, take the following steps:
 
     Like restoring data using BR and Dumpling, when you use Amazon S3 as the storage, there are three methods to grant permissions. The configuration varies with different methods. For details, see [Grant Permissions to Remote Storage](grant-permissions-to-remote-storage.md#aws-account-permissions).
 
-    If you use Ceph or GCS as the storage, you can only grant permissions by importing AccessKey and SecretKey. See [Grant permissions by AccessKey and SecretKey](grant-permissions-to-remote-storage.md#grant-permissions-by-accesskey-and-secretkey).
+    If you use Ceph as the storage, you can only grant permissions by importing AccessKey and SecretKey. See [Grant permissions by AccessKey and SecretKey](grant-permissions-to-remote-storage.md#grant-permissions-by-accesskey-and-secretkey).
```

Comment:
@DanielZhangQD PTAL

Reply:
We should also add a description and link for GCS here.